efficientdet/model_inspect.py

```diff
@@ -40,7 +40,9 @@
 flags.DEFINE_string('logdir', '/tmp/deff/', 'log directory.')
 flags.DEFINE_string('runmode', 'dry', 'Run mode: {freeze, bm, dry}')
 flags.DEFINE_string('trace_filename', None, 'Trace file name.')
-flags.DEFINE_integer('num_classes', 90, 'Number of classes.')
+flags.DEFINE_integer('num_classes', None,
+                     'Number of classes. Defaults to 90 '
```

Comment:
How about deleting num_classes, input_image_size, enable_ema, and data_format (they can simply override with hparams)? WDYT?

Reply:
That would be great. It would be simpler to have just one source of configuration.